### ISSUE TITLE
UWP: impossible to start updates in characteristic

### ISSUE THREAD

**Issue by andrey-val-rodin** [xabre/xamarin-bluetooth-le#618]
## Steps to reproduce

1. Open BLE solution in Visual Studio, set Samples->BLE.Client.UWP (Universal Windows) as startup project and run
2. Connect to device and open characteristic supporting Update
3. Click button Start updates

## Expected behavior
Text "Start updates" below

## Actual behavior
Message box "[StartUpdatesNativeAsync] failed with status: ProtocolError and protocol error Write Not Permitted"

### Crashlog

SystemException "[StartUpdatesNativeAsync] failed with status: ProtocolError and protocol error Write Not Permitted"
Stack:
Plugin.BLE.Extensions.GattResultExtensions.ThrowIfError(GattCommunicationStatus status, String tag, Nullable`1 protocolError)
Plugin.BLE.UWP.Characteristic.<StartUpdatesNativeAsync>d__17.MoveNext()

## Configuration

**Version of the Plugin:** 2.2.0-pre5

**Platform:** Windows 11 Pro build 22000.856

**Device:** ESP32

**Comment by smsissuechecker** [comment]
Hi @andrey-val-rodin,

I'm the friendly issue checker.
Thanks for using the issue template :star2:
I appreciate it very much. I'm sure, the maintainers of this repository will answer, soon.

**Comment by andrey-val-rodin** [comment]
Looks like Windows 11 issue: https://docs.microsoft.com/en-us/answers/questions/962564/wrlc-ble-writeclientcharacteristicconfigurationdes.html

**Comment by janusw** [comment]
So Windows 10 works fine?

**Comment by andrey-val-rodin** [comment]
I don't know

**Comment by andrey-val-rodin** [comment]
I solved issue. The problem was that Windows requires writable Client Characteristic Configuration Descriptor 2902 ("00002902-0000-1000-8000-00805f9b34fb") in characteristic in order to activate updatings: https://stackoverflow.com/a/69126661/8340167. Whereas Android and iOS won't crash if the peripheral doesn't meet the requirements, Windows currently throws exception.

Since I use programmable Bluetooth device (ESP32), I can meet these technical requirements.

But I also faced another challenge: if you add your Bluetooth device in Settings, Windows uses stored device configuration and ignores changes in the device. And for a long time I couldn't figure out why my changes in device didn't work.

Maybe it would be nice to add these findings to Readme.
